src/data/mockUser.js

```diff
@@ -42,12 +42,11 @@ const mockUser = {
         zipcode: "B3 2",
         city: "Birmingham",
       },
+      website: "it works.com",
       contactNumber: "0121 236 8964",
-      description:
-        "A ladies' clothes shop offering a wide range of fashionable pieces.",
+      description: "",
```

Comment:
Did you mean to remove this description?

Reply:
yeah, removed it to check my map function would replace empty descriptions and also thought it would be useful for any other components to highlight potential areas fallbacks would be need.